### ISSUE TITLE
Improvement - design of unavailale chocies 

### ISSUE THREAD

**Issue by NinaHerrmann** [ndunand/moodle-mod_choicegroup#187]
While working on #186, Alex and I were thinking about the design of unavailable choices. As can be seen below, currently, the label is dimmed, and additional information is styled in italics. Another approach would be to use [bootstrap badges](https://getbootstrap.com/docs/4.3/components/badge/) as done in e.g., the `report/status/index.php` table. 
[<image src="https://user-images.githubusercontent.com/18289780/213705959-122da5a4-70dd-4551-9c93-7900407c8f46.png" width="300"/>](https://user-images.githubusercontent.com/18289780/213705959-122da5a4-70dd-4551-9c93-7900407c8f46.png)

[<image src="https://user-images.githubusercontent.com/18289780/213707416-714d1d5b-1a19-4a79-b4e8-8c4b2b02280e.png" width="300"/>](https://user-images.githubusercontent.com/18289780/213707416-714d1d5b-1a19-4a79-b4e8-8c4b2b02280e.png)
Do you think this could be an improvement to the plugin? 
Cheers!

**Comment by ndunand** [comment]
@NinaHerrmann I agree the use of Bootstrap badges would make the interface more legible.
